### ISSUE TITLE
hook_post_db_deploy

### ISSUE THREAD

**Issue by kmoylan** [acquia/cloud-hooks#26]
It'd be great if there were a hook for db_deploy, much like the hook for db_copy.

**Comment by codwazny** [comment]
It's not clear to me what a db_deploy is. Do you mean when you create a new DB role and blank DBs are initialized or something else?

**Comment by kmoylan** [comment]
Sorry, I mean when we re-deploy or restore a database that was previously backed up for a server.  So from the list of database backups, there's a drop-down next to each backup to Download, or Restore, or Delete.  I'd like to be able to hook into when I restore a database backup.  
So maybe db_restore would be a better hook name?

**Comment by bjaspan** [comment]
I agree that a post-db-restore hook sounds like a good idea. I'll create an internal ticket for it.

**Comment by bjaspan** [comment]
What are you thinking you'd use this hook for? What kind of information would you need the hook to receive? For example, you'd certainly want the database name restored to. What about the path of the backup file restored? Or the backup id from Cloud API? etc.

**Comment by kmoylan** [comment]
Basically we just need to know it happened so we can run a couple of tasks.  Any information about which db was restored would be great for logging.

**Comment by paulsanduleac** [comment]
Have any options been explored for post-db-backup?
